### PR TITLE
Merging commits 520 533

### PR DESCRIPTION
**Pull request by ameyachavan007**
includes changes to location page UI
all titles, subtitles, and dropdowns
has improved landing page - state selector card

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.72.156.155:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.72.156.155:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
the location tab is always showing raleigh in the map for every state and not using the state capital dictionary correctly

**Comment by mikahpinegar** [comment]
even when clicking a midwest state the region dropdown shows zone before the county 
zone dropdown

**Comment by mikahpinegar** [comment]
i actually like the old header font better 
old
than the new. can you revert it?
new

**Comment by mikahpinegar** [comment]
i also like the old subtext better 
old_sub
than the new. can you revert it as well?
new_sub

**Comment by mikahpinegar** [comment]
i like the centering and the other changes though. the fonts just feel too aggressive

File: src/pages/Landing/Landing.js [hunk 1]
```diff
@@ -42,27 +49,61 @@ const Landing = () => {
 
   const availableStates = useMemo(() => allStates.map((state) => state.label), [allStates]);
 
+  // Styles for the menu items
```
Comment (mikahpinegar) [inline src/pages/Landing/Landing.js:52]: weird styling
this dropdown behaves weird sometimes like the above screenshot

**Comment by Adamws33** [comment]
> even when clicking a midwest state the region dropdown shows zone before the county zone dropdown

This is still an issue

**Comment by Adamws33** [comment]
> i actually like the old header font better old than the new. can you revert it? new

It looks like the font is still the newer bold.

**Pull request closed without merging**